plugins/flow-visibility/clickhouse-monitor/main.go

```diff
@@ -200,3 +255,37 @@ func getDeleteRowNum(connect *sql.DB) (uint64, error) {
 	deleteRowNum = uint64(float64(count) * deletePercentage)
 	return deleteRowNum, nil
 }
+
+// Parse human readable storage size to number in bytes
+func parseSize(sizeString string) (uint64, error) {
```

Comment:
I am curious: there is no existing function to do that in K8s libraries?

Reply:
Thanks Antonin for reviewing! I took a deeper look into the K8S libraries, and found an API from `resource`. Replaced the function.